VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,50 @@ public class Variables
         [TestMethod]
         public void Declaration()
```

Comment:
This block is for demonstrating Declaration. What would you need to add before the Asserts so that the original Asserts pass _without changing them_?